components/VM_Arm/CMakeLists.txt

```diff
@@ -140,7 +140,9 @@ config_option(
 config_choice(
     VmZynqmpPetalinuxVersion
     ZYNQMP_PETALINUX_VERSION
-    "The version of Petalinux to use"
+    "The version of Petalinux to use. Default is 2018.3, which will images compatible \
+    with the seL4 device tree"
```

Comment:
Something missing in the sentence. Did you mean "use"?

```suggestion
    "The version of Petalinux to use. Default is 2018.3, which will use images compatible \
    with the seL4 device tree"
```

Reply:
Yup, definitely missed a word there